recipes-kernel/linux/files/patches-5.10/0118-net-ethernet-icssg-prueth-Fix-release-of-iep0-on-pro.patch

```diff
@@ -0,0 +1,30 @@
+From 2553ff37e36c63af553b8a06caad4861efd22389 Mon Sep 17 00:00:00 2001
+From: Jan Kiszka <jan.kiszka@siemens.com>
+Date: Tue, 30 Nov 2021 10:53:42 +0100
+Subject: [PATCH 118/118] net: ethernet: icssg-prueth: Fix release of iep0 on
+ probing error
+
+A type caused crashes in case icss_iep_get_idx succeeded for iep0 but
```

Comment:
typo?

Reply:
Yes, a typo of 'typo' - what an irony...